src/main/java/com/jcabi/manifests/Manifests.java

```diff
@@ -187,6 +198,18 @@ public String get(final Object key) {
         return this.attributes.get(key);
     }
 
+    /**
+     * List all values for this key across all appended manifests in the order
+     * they were appended.
+     * @todo: #31 do not expose the internal list structure, or expose readonly
```

Comment:
@westonized let's get rid of this puzzle and just do `return Collections.unmodifiableList(this.multimap.get(key);`